test/dapp/public/main.js

```diff
@@ -1,56 +1,60 @@
-    const counterContract = new ethers.Contract(
-      ContractInfo.address,
-      ContractInfo.abi,
-      provider.getSigner(accounts[0]),
-    );
-    await counterContract.increase();
-    const txSent = document.createElement('div');
-    txSent.id = 'txSent';
-    document.body.appendChild(txSent);
-  });
-
-  const increaseFeesButton = document.querySelector('.increase-fees-button');
-  increaseFeesButton.addEventListener('click', async function () {
-    const accounts = await provider.send('eth_requestAccounts', []);
-    await counterContract.methods.increase().send({ from: accounts[0] });
-    const txSent = document.createElement('div');
-    txSent.id = 'feesTxSent';
-    document.body.appendChild(txSent);
-  });
+  let counterContract;
+  let accounts;
 
   const connectButton = document.querySelector('.connect-button');
   connectButton.addEventListener('click', async function () {
-    const accounts = await ethereum.request({
+    accounts = await ethereum.request({
       method: 'eth_requestAccounts',
     });
+    counterContract = new ethers.Contract(ContractInfo.address, ContractInfo.abi, provider.getSigner(accounts[0]));
     const connected = document.createElement('div');
     connected.id = 'connected';
+    connected.textContent = 'connected';
     document.body.appendChild(connected);
   });
 
+  const increaseButton = document.querySelector('.increase-button');
+  increaseButton.addEventListener('click', async function () {
+    await counterContract.increase({ from: accounts[0] });
+    const increase = document.createElement('div');
+    increase.id = 'increased';
+    increase.textContent = 'increased';
+    document.body.appendChild(increase);
+  });
+
+  const increaseFeesButton = document.querySelector('.increase-fees-button');
+  increaseFeesButton.addEventListener('click', async function () {
+    await counterContract.increase({ from: accounts[0] });
+    const increaseFees = document.createElement('div');
+    increaseFees.id = 'increasedFees';
+    increaseFees.textContent = 'increasedFees';
+    document.body.appendChild(increaseFees);
+  });
+
   const signButton = document.querySelector('.sign-button');
   signButton.addEventListener('click', async function () {
     const accounts = await provider.send('eth_requestAccounts', []);
     const signer = provider.getSigner(accounts[0]);
     await signer.signMessage('TEST');
     const signed = document.createElement('div');
     signed.id = 'signed';
+    signed.textContent = 'signed';
     document.body.appendChild(signed);
   });
 
   const transferButton = document.querySelector('.transfer-button');
   transferButton.addEventListener('click', async function () {
     const accounts = await provider.send('eth_requestAccounts', []);
-    await counterContract.methods.increase().send();
-    await web3.eth.sendTransaction({ to: accounts[0], from: accounts[0], value: web3.utils.toWei('0.01') });
+    // await counterContract.increase().send();
```

Comment:
comment can be removed

Reply:
this should fixed in https://github.com/TenKeyLabs/dappwright/pull/29 as well